inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -501,12 +501,28 @@ void GNAGraphCompiler::PowerPrimitive(InferenceEngine::CNNLayerPtr layer) {
     auto input = layer->insData[0].lock();
 
     auto outputs = *layer->outData.begin();
-
     uint32_t num_rows_in = FROM_IR_DIM(input, 1);
     uint32_t num_columns_in = FROM_IR_DIM(input, 2);
     uint32_t num_rows_out = num_rows_in;
     uint32_t num_padding = ALIGN(num_rows_in, 8) - num_rows_in;
 
+    if (num_columns_in > 8) {
+        auto PrevFunctionalLayer = [](CNNLayerPtr l, int idx = 0) {
```

Comment:
Why do you put CNNNetPrevLayerSkipCertain + gnalog in a lambda and invoke it once? Is it necessary?